web/src/search/input/interactive/FilterInput.tsx

```diff
@@ -297,110 +359,166 @@ export class FilterInput extends React.Component<Props, State> {
     private downshiftItemToString = (suggestion?: Suggestion): string => (suggestion ? suggestion.value : '')
 
     public render(): JSX.Element | null {
+        const isText = isTextFilter(this.props.filterType)
+
+        return (
+            <div
+                className={`filter-input ${
+                    this.props.editable && isText
+                        ? `${this.props.isHomepage ? 'filter-input--active-homepage' : 'filter-input--active'}`
```

Comment:
Even without the redundant template string, readability could be improved here. Maybe extract some of this logic in a variable above?

```suggestion
                        ? {this.props.isHomepage ? 'filter-input--active-homepage' : 'filter-input--active'
```